packages/next/build/webpack/require-hook.ts

```diff
@@ -126,6 +126,8 @@ const hookPropertyMap = new Map(
       'next/dist/compiled/@babel/runtime/package.json',
     ],
     ['node-fetch', 'next/dist/compiled/node-fetch'],
+    ['styled-jsx/style', 'styled-jsx/style'],
+    ['styled-jsx', 'styled-jsx'],
```

Comment:
Should these be pointing to `require.resolve('styled-jsx/style')` instead? Currently this seems like a no-op

Reply:
They'll be mapped to the path from `require.resolve` 

`.map(([request, replacement]) => [request, require.resolve(replacement)])`